### ISSUE TITLE
`source` parameter unable to handle files with unicode content

### ISSUE THREAD

**Issue by 9ao9ai9ar** [mikitex70/plantuml-markdown#56]
## Environment

Windows 10
Python 3.9.6
mkdocs 1.2.2
plantuml-markdown 3.4.2

## Test cases

`test1.puml`:

```plantuml
@startuml actor
' 各
actor me
rectangle box {
  me --> (item)
}
@enduml
```

`test2.puml`:

```plantuml
@startuml actor
actor 我
rectangle 方框 {
  我 --> (項目)
}
@enduml
```

## Output

    ```plantuml source="test1.puml"
    ```

```
  File "C:\Users\9ao9ai9ar\miniconda3\envs\dev\lib\site-packages\plantuml_markdown.py", line 176, in _replace_block
    code += f.read()
  File "C:\Users\9ao9ai9ar\miniconda3\envs\dev\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 20: character maps to <undefined>
```

    ```plantuml source="test2.puml"
    ```

```
  File "C:\Users\9ao9ai9ar\miniconda3\envs\dev\lib\site-packages\plantuml_markdown.py", line 280, in _render_remote_uml_image
    return PlantUML("%s/%s/" % (self.config['server'], img_format)).processes(plantuml_code)
  File "C:\Users\9ao9ai9ar\miniconda3\envs\dev\lib\site-packages\plantuml.py", line 173, in processes
    raise PlantUMLHTTPError(response, content)
  File "C:\Users\9ao9ai9ar\miniconda3\envs\dev\lib\site-packages\plantuml.py", line 56, in __init__
    if not self.message:
AttributeError: 'PlantUMLHTTPError' object has no attribute 'message'
```

The test cases render fine if the code is inlined; this problem only manifests itself when using the `source` parameter.

**Comment by mikitex70** [comment]
I've release the version `3.4.3`, give it a try.
I've forced `utf-8` encoding for files referenced by the `source` parameter; this should resolve your issue.
A new configuration option (`encoding`) can be used to change the default, useful in Windows which has `cp1252` as the default encoding.

**Comment by 9ao9ai9ar** [comment]
I can confirm that this issue is fixed in version `3.4.3`. Thank you for the prompt response and release of a fix.
